src/core/basic_client.rs

```diff
@@ -722,6 +724,95 @@ impl BasicClient {
         Ok(())
     }
 
+    /// **[Cryptographic Operation]** Create an asymmetric signature on a message.
+    ///
+    /// The key intended for signing **must** have its `sign_hash` flag set
```

Comment:
Maybe `sign_message` here? Same for the verify_message

Reply:
Corrected - thanks for pointing.